connectors/sources/tests/fixtures/aws/load_storage.py

```diff
@@ -0,0 +1,31 @@
+import boto3
+import sys
+
+BUCKET_NAME = "ent-search-ingest-dev"
+OBJECT_COUNT = 15000
+ENDPOINT_URL = "http://127.0.0.1"
+
+
+def main():
+    """Method for generating 15k document for aws s3 emulator"""
+    try:
+        try:
+            PORT = sys.argv[1]
+        except IndexError:
+            PORT = 5000
+        s3_client = boto3.client("s3", endpoint_url=f"{ENDPOINT_URL}:{PORT}")
+        s3_client.create_bucket(Bucket=BUCKET_NAME)
+        print("Creating objects on the aws-moto server")
+        for object_id in range(0, OBJECT_COUNT):
+            s3_client.put_object(
+                Key=f"{BUCKET_NAME}/{object_id}",
+                Bucket=BUCKET_NAME,
+                Body=f"Testing object{object_id} document for bucket: {BUCKET_NAME}",
```

Comment:
let's add bigger bodies too, like what was done with NetworkDrive so we can make sure chunk downloads etc. are exercised and we can run the ftest realistically to very the memory does not blow